{"name": "Class ClosedByInterruptException", "module": "java.base", "package": "java.nio.channels", "text": "Checked exception received by a thread when another thread interrupts it\n while it is blocked in an I/O operation upon a channel.  Before this\n exception is thrown the channel will have been closed and the interrupt\n status of the previously-blocked thread will have been set.", "codes": ["public class ClosedByInterruptException\nextends AsynchronousCloseException"], "fields": [], "methods": []}